contacts/tests.py

```diff
@@ -1,3 +1,123 @@
-# from django.test import TestCase
+import pytest
+from users.models import User, Seeker
+from apartments.models import Apartment, City
+from .models import Connection
+from django.db import transaction, IntegrityError
 
-# Create your tests here.
+
+@pytest.fixture
+def new_city():
+    city = City(cityName='Tel Aviv')
+    city.save()
+    return city
+
+
+@pytest.fixture
+def seeker_1(new_city):
+    user1 = create_new_user("t1@m.com", "test1", "test", "1995-05-05", "testing")
+    seeker = create_seeker(user1, new_city, "2020-05-05", 100, 3000, 2, 2, "Hi")
+    seeker.save()
+    return seeker
+
+
+@pytest.fixture
+def seeker_2(new_city):
+    user2 = create_new_user("t2@m.com", "test2", "test", "1995-05-05", "testing")
+    seeker = create_seeker(user2, new_city, "2020-05-05", 300, 3000, 3, 3, "Hi")
+    seeker.save()
+    return seeker
+
+
+@pytest.fixture
+def apartment_1(new_city):
```

Comment:
If there's no `apartment_2` or something of that sort (and applying my former comment should remove the need for those), please name the fixtures in a better way (this comment applies to all of them).

Reply:
Thanks, I renamed the fixtures and marking this as resolved!